buildspec.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
```

Comment:
Can we add a
```yml
# Paragraph up here explaining
# what this file is for
```
(It's clear right now during PR review, but may not be in other contexts / for other readers.)

Does the `buildspec.yml` work generically accross multiple systems, or is this specific to CodeBuild? Can you add in the comment?

Reply:
I've added some information about the file, including a couple links to the AWS docs in case the reader is not familiar with the product. One other thing we can do is to move this file out of here and make it part of the Cloudformation template generated in the `amplify-ci-support` repo.